electric-operator.el

```diff
@@ -889,7 +889,8 @@ Also handles C++ lambda capture by reference."
   (defun electric-operator-python-mode-kwargs-= ()
     (cond
      ((electric-operator-python-mode-in-lambda-args?) "=")
-     ((eq (electric-operator-enclosing-paren) ?\() "=")
+     ((and (eq (electric-operator-enclosing-paren) ?\()
+           (not (electric-operator-looking-back-locally ":[^,(]*"))) "=")
```

Comment:
I think this will fail on: `def foo(a: Tuple[int, int, int]=`, can you add a test for that case?

I'm not sure how to handle this, we need to "step over" any paired `[]` without looking inside it. This is something that classically regexes can't do, but maybe emacs' regexes can...